tests/core/engines/alpha/steps/events.py

```diff
@@ -193,6 +194,22 @@ def given_a_flagged_customer_message(
     return session.id
 
 
+@step(
+    given,
+    parsers.parse('an utterance with action of "{action}" and reason of "{reason}"'),
+)
+def given_an_utterance(
```

Comment:
```@step(..., 'an utterance request "{action}", to {reason}')
def given_an_utterance_request```

where in the BDD "reason" shouldn't be in quotes, and should be either "follow up with the customer" or "buy time"

Here's what usage would look like:
`Given an utterance request "Ask if they got your message", to follow up with the customer`
`Given an utterance request "Say you'll be late", to buy time`